### PR TITLE
GetUserCompletedGames API

### PR DESCRIPTION
**Pull request by bdjeffyp**
A new public facing API to return the results of `getUsersCompletedGamesAndMax()`. This can allow external apps to see the number of games a user has played, the number of games the user has completed and mastered, as well as details on completion percentage for each game.

- Updated the APIDemo page to include the new API call.
- Updated the README to include some details on using XAMPP and updating the included version of PHP to 7.4, as well as creating a dummy database.

### CODE REVIEW

**Comment by bdjeffyp** [comment]
Just wanting to ping this... Any chance of someone taking a look? I'd appreciate it! If there is any issues with it, I'll be happy to make some changes. Just let me know. Thanks!

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ It is a "flat" PHP project; individual php files within the `public` folder hand
 
 **[XAMPP](https://www.apachefriends.org/download.html)** provides an easy way to run an Apache web server, MySQL/MariaDB, and PHP on your system.
 
+- Note that XAMPP comes packaged with PHP 7.2. You will need to update to PHP 7.4 or errors will occur when you open the application. To update PHP, do the following:
```
Comment (meleu) [inline README.md:16]: I suggest moving these instructions to the wiki and here on the README just put a note saying something like "use PHP 7.4+ (check the wiki if you're having problems)"

Comment (bdjeffyp) [reply README.md:16]: That makes sense. I will make that change. Thanks!

**Comment by bdjeffyp** [comment]
How does it look now, @meleu?

**Review by meleu — APPROVED** [review]

**Pull request merged**